src/SAWScript/Interpreter.hs

```diff
@@ -3836,11 +3837,212 @@ primitives = Map.fromList
     ---------------------------------------------------------------------
     -- Crucible/MIR commands
 
+  , prim "mir_alloc" "MIRType -> MIRSetup SetupValue"
+    (pureVal mir_alloc)
+    Experimental
+    [ "Declare that an immutable reference to the given type should be allocated"
+    , "in a MIR specification. Before `mir_execute_func`, this states that"
+    , "the function expects the object to be allocated before it runs."
+    , "After `mir_execute_func`, it states that the function being"
+    , "verified is expected to perform the allocation."
+    ]
+
+  , prim "mir_alloc_mut" "MIRType -> MIRSetup SetupValue"
+    (pureVal mir_alloc_mut)
+    Experimental
+    [ "Declare that a mutable reference to the given type should be allocated"
+    , "in a MIR specification. Before `mir_execute_func`, this states that"
+    , "the function expects the object to be allocated before it runs."
+    , "After `mir_execute_func`, it states that the function being"
+    , "verified is expected to perform the allocation."
+    ]
+
+  , prim "mir_assert" "Term -> MIRSetup ()"
+    (pureVal mir_assert)
+    Experimental
+    [ "State that the given predicate must hold.  Acts as `mir_precond`"
+    , "or `mir_postcond` depending on the phase of specification in which"
+    , "it appears (i.e., before or after `mir_execute_func`."
```

Comment:
```suggestion
    , "it appears (i.e., before or after `mir_execute_func`)."
```

Reply:
Well spotted. What's more, this same mistake is present in the docstrings for `llvm_assert` and `jvm_assert`, from where I originally copied this! I've pushed a separate commit which fixes all three copies of the mistake.